stix_shifter/stix_translation/src/modules/security_advisor/security_advisor_utils.py

```diff
@@ -0,0 +1,42 @@
+class FileHashLookupException(Exception):
```

Comment:
I guess you copied this from the QRadar adapter. There is code here that is very QRadar dependent (`_find_hash_type_by_logsource`). I think the only part you would need is the hash_type_by_length method. I'd recommend just creating a small helper method in this file, or put the method right in the `security_advisor_translator.py` file.

Reply:
Another option would be to create a new helper class under the `stix_translation/src/utils` directory so others can use it if needed.